### DISCUSSION TITLE
How can we make it HIPAA comliant?

### DISCUSSION THREAD

**Discussion by SentrixMediaCorp** [docusealco/docuseal#243] [Ideas]
We are in a clinical setting where in our clinic we need to sign documents between departments mainly for SOPs. how can docusign be HIPAA compliant?

**Comment by omohokcoj** [comment]
Hi @SentrixMediaCorp DocuSeal hosted on-premises gives you the control over your data needed to pass the compliance check by the HIPAA auditor

**Comment by calmond01** [comment]
This [post](https://www.formhippo.com/is-docusign-hipaa-compliant/) goes into detail about DocuSign and HIPAA compliance.  Or you can use a platform like [FormHippo Signatures.](https://www.formhippo.com/formhipposignatures/)
